core/lexicon/en/topmenu.inc.php

```diff
@@ -30,19 +30,19 @@
 $_lang['edit_menu'] = 'Menus';
 $_lang['edit_menu_desc'] = 'Manage the top Manager menu';
 $_lang['eventlog_viewer'] = 'Error Log';
-$_lang['eventlog_viewer_desc'] = 'View the MODX error.log.';
+$_lang['eventlog_viewer_desc'] = 'View the MODX error log';
 $_lang['export_site'] = 'Export Static HTML';
-$_lang['export_site_desc'] = 'Export the current site into static HTML pages.';
+$_lang['export_site_desc'] = 'Export the current site into static HTML pages';
 $_lang['file_browser'] = 'Media Browser';
 $_lang['file_browser_desc'] = 'View, upload and manage media';
 $_lang['flush_access'] = 'Flush Your Permissions';
-$_lang['flush_access_confirm'] = 'Are you sure you want to flush your access permissions? NOTE: This does not affect other user sessions.';
+$_lang['flush_access_confirm'] = 'Are you sure you want to flush your access permissions? NOTE: This does not affect other user sessions';
```

Comment:
all `_confirm` entries should use a dot at the end, since these contain full sentences.

Reply:
Yes, it makes sense. Although error messages also come with a dot and without.
@JoshuaLuckers what do you think?

Reply:
I think @Jako has a good point for entries that are a full sentence.